ArduCopter/version.cpp

```diff
@@ -0,0 +1,16 @@
+#include "version.h"
+
+#include "GCS_MAVLink/GCS.h" // for FIRMWARE_VERSION_TYPE_DEV and friends
+
+const AP_FWVersion fwver
```

Comment:
@peterbarker I don't see why making this a member of Copter makes you need or want it to be a class? By the way, a struct is kind of a class and can have constructors too.

I just don't understand why this is global but then there is a method in GCS_MAVLINK to get it. If it is global you can just access it, you don't need a method to retrieve it from anywhere.

I would still say it should be a member of the vehicle, I hate global variables in a C++ world.